.gitmodules

```diff
@@ -1,3 +1,12 @@
 [submodule "contracts/lib/eigenlayer-middleware"]
 	path = contracts/lib/eigenlayer-middleware
 	url = git@github.com:Layr-Labs/eigenlayer-middleware.git
+[submodule "services/operatorsinfo/integration_test_deployment/lib/forge-std"]
+	path = services/operatorsinfo/integration_test_deployment/lib/forge-std
+	url = https://github.com/foundry-rs/forge-std
+[submodule "services/operatorsinfo/integration_test_deployment/lib/eigenlayer-middleware"]
+	path = services/operatorsinfo/integration_test_deployment/lib/eigenlayer-middleware
+	url = https://github.com/Layr-Labs/eigenlayer-middleware
+[submodule "services/operatorsinfo/integration_test_deployment/lib/eigenlayer-middleware-offchain"]
+	path = services/operatorsinfo/integration_test_deployment/lib/eigenlayer-middleware-offchain
+	url = https://github.com/Layr-Labs/eigenlayer-middleware-offchain
```

Comment:
are these still needed? don't think they are used anymore after the last commit that deleted a bunch of stuff?